__tests__/app.test.js

```diff
@@ -228,3 +228,90 @@ describe('POST /api/articles/:article_id/comments', () => {
     expect(res.body.message).toBe('bad request');
   });
 });
+
+describe('GET /api/articles queries', () => {
+  test('200: sorted by date descending as default', async () => {
+    const res = await request(app).get('/api/articles').expect(200);
+    expect(res.body).toBeSortedBy('created_at', { coerce: true });
+  });
+  test('200: sorted by any given column in descending order as default', async () => {
+    const res = await request(app)
+      .get('/api/articles?sort_by=article_id')
+      .expect(200);
+    expect(res.body).toBeSortedBy('article_id', {
+      coerce: true,
+      descending: true,
+    });
+    const res2 = await request(app)
+      .get('/api/articles?sort_by=comment_count')
+      .expect(200);
+    expect(res2.body).toBeSortedBy('comment_count', {
+      coerce: true,
+      descending: true,
+    });
+  });
+  test('200: sorted by any given column in choice of descending/ascending order', async () => {
+    const res = await request(app)
+      .get('/api/articles?sort_by=article_id&order=asc')
+      .expect(200);
+    expect(res.body).toBeSortedBy('article_id', {
+      coerce: true,
+      ascending: true,
+    });
+    const res2 = await request(app)
+      .get('/api/articles?sort_by=comment_count&order=asc')
+      .expect(200);
+    expect(res2.body).toBeSortedBy('comment_count', {
+      coerce: true,
+      ascending: true,
+    });
+  });
+  test('200: returns in order of choice sorted by created_at when given no column', async () => {
```

Comment:
Could you please rephrase this to reference "sorted_by" instead of the column specifically? We shouldn't start giving away clues about our database in our tests.